x-pack/legacy/plugins/apm/public/components/shared/TransactionBreakdown/index.tsx

```diff
@@ -38,32 +37,76 @@ const NoTransactionsTitle = styled.span`
   font-weight: bold;
 `;
 
-const TransactionBreakdown: React.FC = () => {
-  const [showChart, setShowChart] = useState(false);
+const TransactionBreakdown: React.FC<{
+  initialIsOpen?: boolean;
+}> = ({ initialIsOpen }) => {
+  const [showChart, setShowChart] = useState(!!initialIsOpen);
 
   const {
     data,
     status,
     receivedDataDuringLifetime
   } = useTransactionBreakdown();
 
+  const total = data ? data.total : undefined;
```

Comment:
This naming "total" is a bit misleading. In other places we use `total` as the counter (eg. `hits.total` is the response from es).

At least when looking through this code I was expecting `total` to be an int until I saw the iteration below.

Reply:
Agreed. What about `overview`?

Reply:
Can we make it something plural to indicate it's an array?

Reply:
Yeah, what about `kpis`?

Reply:
That works 👍

Reply:
changed!